js/components/authoring.tsx

```diff
@@ -43,19 +45,16 @@ const VIEW_OPTIONS: Option[] = [
 const SKIPPED_OPTIONS: Option[] = ["authoring", "planetWizard", "planetWizardSteps", "sidebar", "preset", "modelId", "densityWordInPlanetWizard"];
 
 // All the options manually defined in various sections.
-const CUSTOM_OPTIONS: Option[] = [];
-CUSTOM_OPTIONS.concat(MAIN_OPTIONS, VIEW_OPTIONS, SKIPPED_OPTIONS).map(opt => typeof opt === "string" ? opt : opt[0]);
+const CUSTOM_OPTIONS: Option[] = [].concat(MAIN_OPTIONS as any, VIEW_OPTIONS as any, SKIPPED_OPTIONS as any)
```

Comment:
Would this work as a means of eliminating the `any`s?
```suggestion
const CUSTOM_OPTIONS: Option[] = [...MAIN_OPTIONS, ...VIEW_OPTIONS, ...SKIPPED_OPTIONS]
```

Reply:
It helped actually, thanks. :) Weirdly, even without the `.map()` call TS wasn't complaining about types in this case, but it's a nicer approach anyway.